tools/PackModules.ps1

```diff
@@ -0,0 +1,141 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+#This script will pack build artifacts under temporary folder "artifacts/tmp" and output Az.*.nupkg to "artifacts"
+
+param(
+    [Parameter(Mandatory = $false, Position = 1)]
+    [string]$Artifacts,
+
+    [Parameter(Mandatory = $false, Position = 5)]
+    [string]$NugetExe
+)
+
+<#################################################
+#
+#               Helper functions
+#
+#################################################>
+
+<#
+.SYNOPSIS Write out to a file using UTF-8 without BOM.
+.PARAMETER File
+The file to write the contents too.
+.PARAMETER Text
+The new file contents.
+#>
+function Out-FileNoBom {
+    param(
+        [System.string]$File,
+        [System.string]$Text
```

Comment:
nit: suggest to use string for consistent style

Reply:
Resolved